<?xml version="1.0"?>
<launch>
    <group ns="front_cam">
        <rosparam file="$(find ip_camera)/config/ipcam_1.yaml" command="load"/>
    </group>

    <node name="rosbag" pkg="rosbag" type="play" args="$(find perception)/dataset/dataset3.bag" />
    <node name="perception" pkg="perception" type="perception_single" output="screen">
        <param name="image_topic_prefix" value="/carla/camera_"/>
        <param name="batch_size" type="int" value="1"/>
    </node>
</launch>